storage/tests/system.py

```diff
@@ -142,6 +138,47 @@ def test_bucket_update_labels(self):
         bucket.update()
         self.assertEqual(bucket.labels, {})
 
+    @unittest.skipUnless(USER_PROJECT, 'USER_PROJECT not set in environment.')
```

Comment:
Explicit `None`-check?

Reply:
Don't care:  if it is set to an empty string, I still don't want to run the test.